config/application.rb

```diff
@@ -136,6 +137,12 @@ class Application < Rails::Application
     # Add apidoc hash in headers for smarter caching
     config.middleware.use "Apipie::Middleware::ChecksumInHeaders"
 
+    Foreman::Logging.configure(:log_directory => "#{Rails.root}/log", :environment => Rails.env)
+    config.colorize_logging = Foreman::Logging.config[:colorize]
```

Comment:
Yea, I think this would obsolete it as we would want to control it from a central configuration for logging. I will update these instances, but I assume I will also need to update the puppet module once we settle on the overall configuration setup.